### PR TITLE
[DDAN-34] add submitter, master unittest 

### PR DESCRIPTION
**Pull request by Armstrong-SuHwan**
### PURPOSE OF PR
* Simple summary of this PR
- add submitter, master, process monitor unit test code

### What type of PR is it?
* fix / feature / improvement /  hot-fix / refactoring

### JIRA ISSUE
* https://ddangcoonfire.atlassian.net/browse/DDAN-3

### HAVE YOU CHECKED ...
* [x] ADD JIRA ID ON PR TITLE (ex. [DDAN-1234] PR TITLE)
* [x] pre-commit
* [x] Branch Format ( type/issue-NUM ) (ex. refactoring/DDAN-1234 )

### Explanation
( Adding screenshot would be great help )

### CODE REVIEW

File: experiment_scheduler/submitter/execute.py [hunk 1]
```diff
@@ -29,14 +29,13 @@ def main():
 
     with open(file_path) as f:
         parsed_yaml = yaml.load(f, Loader=yaml.FullLoader)
-
-    channel = grpc.insecure_channel('localhost:50050')
+    channel = grpc.insecure_channel('localhost:50052')
```
Comment (ptaejoon) [inline experiment_scheduler/submitter/execute.py:32]: Set hard_coded channel address to cfg path.
Example codes are in master.py

File: experiment_scheduler/submitter/exs.py [hunk 2]
```diff
@@ -40,7 +40,9 @@ def main():
     Select and execute a function from the command list.
     """
     name = parse_args().operation
+    print(name)
```
Comment (ptaejoon) [inline experiment_scheduler/submitter/exs.py:43]: set print to logger.info or delete it

File: experiment_scheduler/submitter/exs.py [hunk 3]
```diff
@@ -40,7 +40,9 @@ def main():
...
     del sys.argv[1]
+    print(COMMAND_LIST[name])
```
Comment (ptaejoon) [inline experiment_scheduler/submitter/exs.py:45]: same here

File: experiment_scheduler/submitter/execute.py (hunk 1 above)
Comment (ptaejoon) [inline experiment_scheduler/submitter/execute.py:32]: set channel address using cfg file

**Review by ddangcoonfire — APPROVED** [review]
confirmed changes about requirements

**Pull request merged**